lib/core/hardwrap.dart

```diff
@@ -1,3 +1,19 @@
+/*
+Copyright 2020-2021 Vishesh Handa <me@vhanda.in>
```

Comment:
Sorry, but you'll need to put your name.

Reply:
I should have been more specific in the instructions :(

Reply:
I'm really sorry. I was completely misunderstood.
Thank you for your patience.